docs/use-cases/experiments.md

```diff
@@ -0,0 +1,316 @@
+| treatment2 | Hello there | Sign up         |
+| treatment2 | Hello there | Get started     |
+
+Instead of creating a separate feature per element, we can create a single feature for the Hero section and define multiple variables for each element.
+
+The relationship can be visualized as:
+
+- one **feature**
+- having multiple **variations**
+- each variation having its own set of **variable values**
+
+```yml
+# features/hero.yml
+description: Hero section
+tags:
+  - all
+
+bucketBy: deviceId
+
+defaultVariation: control
+
+# define a schema of all variables
+# scoped under `hero` feature first
+variablesSchema:
+  - key: headline
+    type: string
+    defaultValue: Welcome
+
+  - key: ctaButtonText
+    type: string
+    defaultValue: Sign up
+
+variations:
+  - value: control
+    weight: 25
+
+  - value: treatment1
+    weight: 25
+    variables:
+      # we only define variables inside variations,
+      # if the values are different than the default values
+      - key: ctaButtonText
+        value: Get started
+
+  - value: treatment2
+    weight: 25
+    variables:
+      - key: headline
+        value: Hello there
+
+  - value: treatment3
+    weight: 25
+    variables:
+      - key: headline
+        value: Hello there
+
+      - key: ctaButtonText
+        value: Get started
+
+environments:
+  production:
+    rules:
+      - key: "1"
+        segments: "*"
+        percentage: 100
+```
+
+We just set up our first Multivariate test experiment that is:
+
+- rolled out to 100% of our traffic to everyone
+- with an even 25% split among all its variations
+- with each variation having different values for the variables
+
+## Evaluating variables
+
+In your application, you can access the variables of the `hero` feature as follows:
+
+```js
+const featureKey = "hero";
+const attributes = { deviceId: "device-123" };
+
+const headline = sdk.getVariable(featureKey, "headline", attributes);
+const ctaButtonText = sdk.getVariable(featureKey, "ctaButtonText", attributes);
+```
+
+Use the values inside your hero element (component) when you render it.
+
+## Tracking
+
+We understood how to create features for defining simple A/B tests and also more complex multivariates using variables in Featurevisor, and then evaluate them in the runtime in our applications when we need those values.
+
+But we also need to track the performance of our experiments to understand which variation is doing better than others.
+
+This is where the `activate()` method of the SDK comes in handy. Before we call the method, let's first set up our activation event handler in the SDK initialization:
+
+```js
+import { createInstance } from "@featurevisor/sdk";
+
+const sdk = createInstance({
+  datafile: "https://cdn.yoursite.com/datafile.json",
+
+  onReady: () => console.log("Datafile has been fetched and SDK is ready"),
+
+  onActivation: (featureKey, variation, attributes, captureAttributes) => {
+    // send the event to your analytics platform
```

Comment:
I think it would be good to have an example of how to send data to some of the main analytics platforms - GA4, Adobe etc.

Reply:
added a link to the Google Tag Manager example in this section.